README.md

```diff
@@ -54,50 +53,63 @@ spec:
+I0223 07:33:59.500418 1 nodemaintenance_controller.go:449] evicting pod openshift-network-diagnostics/network-check-source-865d4b5578-n2cxg
+I0223 07:33:59.500790 1 nodemaintenance_controller.go:449] evicting pod openshift-ingress/router-default-7548cf6fb5-rgxrq
+I0223 07:33:59.500944 1 nodemaintenance_controller.go:449] evicting pod openshift-marketplace/12a4cfa0c2be01867daf1d9b7ad7c0ae7a988fd957a2ad6df0d72ff6875lhcx
+I0223 07:33:59.501061 1 nodemaintenance_controller.go:449] evicting pod openshift-marketplace/nmo-downstream-8-8nms7
 ...
 ```
 
 ### Set Maintenance off - Delete the NodeMaintenance CR
 
-To remove maintenance from a node, delete the corresponding `NodeMaintenance` CR:
+To remove maintenance from a node, delete the corresponding `NodeMaintenance` (or `nm` which is a shortName) CR:
 
 ```sh
-$ kubectl delete nodemaintenance nodemaintenance-sample
-
+$ kubectl delete nm nodemaintenance-sample
+nodemaintenance.nodemaintenance.medik8s.io "nodemaintenance-sample" deleted
 $ kubectl logs <nmo-pod-name>
-{"level":"info","ts":1551794725.0018933,"logger":"controller_nodemaintenance","msg":"Reconciling NodeMaintenance","Request.Namespace":"default","Request.Name":"node02"}
-{"level":"info","ts":1551794725.0021605,"logger":"controller_nodemaintenance","msg":"NodeMaintenance Object: default/node02 Deleted ","Request.Namespace":"default","Request.Name":"node02"}
-{"level":"info","ts":1551794725.0022023,"logger":"controller_nodemaintenance","msg":"uncordon Node: node02"}
-
+2022-02-24T14:27:35.332Z INFO controller-runtime.manager.controller.nodemaintenance Reconciling NodeMaintenance {"reconciler group": "nodemaintenance.medik8s.io", "reconciler kind": "NodeMaintenance", "name": "nodemaintenance-sample", "namespace": ""}
+time="2022-02-24T14:27:35Z" level=info msg="Maintenance taints will be removed from node node02"
+time="2022-02-24T14:27:35Z" level=info msg="Applying medik8s.io/drain taint remove on Node: node02"
+...
 ```
 
 ## NodeMaintenance Status
 
-The NodeMaintenance CR can contain the following status fields:
+The `NodeMaintenance` CR can contain the following status fields:
 
 ```yaml
+$ kubectl get nm nodemaintenance-sample -o yaml
 apiVersion: nodemaintenance.medik8s.io/v1beta1
 kind: NodeMaintenance
 metadata:
-  name: nodemaintenance-xyz
+  creationTimestamp: "2022-02-24T14:37:25Z"
```

Comment:
I don't know why some of the fields were missing. Maybe it was intended as a "demo" `NodeMaintenance` status.

Reply:
That would be my assumption.  Only showing what is important